README.md

```diff
@@ -0,0 +1,155 @@
+
+You need to make a few configuration changes to support `DPDK`.
+
+`DPDK` needs a small kernel module to set up the device, map device memory to user-space and register interrupts. The standard `uio_pci_generic` module included in the Linux kernel can provide the capability. To load the module, use the command,
+
+```
+host$ sudo modprobe uio_pci_generic
+```
+
+Debian and CentOS by default include this module. If you are on an Ubuntu host, `uio_pci_generic` is not part of the base system. You will need to install an additional package,
+
+```
+sudo apt install linux-modules-extra-$(uname -r)
+```
+
+DPDK also needs `HugePages` support for the large memory pool allocation used for packet buffers. To enable `HugePages`, use the following commands,
+
+```
+host$ sudo su
+host$ echo "vm.nr_hugepages = 2048" >> /etc/sysctl.conf
+host$ sysctl -e -p
+host$ exit
+```
+
+Before your `Capsule` application can access a network interface on the host, the interface must be bound to a DPDK compatible driver with the [`dpdk-devbind`](https://doc.dpdk.org/guides/tools/devbind.html) utility. To bind the driver, find the interface's PCI address and use the command,
+
+```
+host$ docker pull getcapsule/dpdk-devbind:18.11.6
+host$ docker run --rm --privileged --network=host \
+    -v /lib/modules:/lib/modules \
+    getcapsule/dpdk-devbind:18.11.6 \
+    /bin/bash -c 'dpdk-devbind.py --force -b uio_pci_generic #PCI_ADDR#'
+```
+
+Once you've made the necessary changes, you can pull down the sandbox container and run it,
+
+```
+host$ docker pull getcapsule/sandbox:18.11.6-1.42
+host$ docker run ...
+```
+
+### Without Docker
+
+If you choose not to use `Docker`, or if you are using a Linux distribution incompatible with the `Debian` based sandbox, then you need to [install DPDK from source](https://doc.dpdk.org/guides/linux_gsg/build_dpdk.html) yourself.
+
+In additional to the [required tools and libraries](https://doc.dpdk.org/guides/linux_gsg/sys_reqs.html#compilation-of-the-dpdk), you also need the library for packet captures to run some `Capsule` example applications. Install `libpcap-dev` for `Debian`/`Ubuntu` or `libpcap-devel` for `RHEL`/`CentOS`/`Fedora`.
+
+Once you have installed all the necessary tools and libraries on your system, you should use our [script](scripts/dpdk.sh) to install the version of DPDK that `Capsule` uses.
+
+You can also use another [script](scripts/rustup.sh) to install the latest stable `Rust` if you don't have that setup already. Since kernel 4.0, running `DPDK` applications requires `root` privileges. You must install the `Rust` toolchain for the `root` user as well if you want to run your project with `cargo run`.
+
+We've provided another Vagrant virtual machine for development sans `Docker` if you don't want to manually install `DPDK`.
```

Comment:
`We've also provided`

Reply:
"for development, sans `Docker`, if you don't want to manually install `DPDK`, `Rust`, and the necessary packages."